src/main/java/APIInterface/BulkStockAPIResponse.java

```diff
@@ -0,0 +1,19 @@
+package APIInterface;
+
+import java.util.HashMap;
+
+public class BulkStockAPIResponse {
+    /**
+     * Response model class containing output from a bulk price request to the
+     * stock price API.
+     */
+    private HashMap<String, Double> symbolToPrice;
```

Comment:
Should be type map as HashMap is just a specific implementation

Reply:
Fixed!